resources/generate_book.py

```diff
@@ -413,13 +413,19 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
-            # todo: overview of water, thirst and drinking
-            # todo: overview of player nutrition and it's effects
-            # todo: introduction to different food sources, links to preservation
+        entry('food_and_water', 'Food and Water', 'tfc:food/orange', pages=(
+            text('In TerraFirmaCraft, not only must you manage your hunger, but you must manage your thirst. Hunger works similar to vanilla. Most pieces of food restore about a fifth of your hunger bar. Some pieces of food may restore a bit less, such as $(thing)Cattail Roots$(). Eating food also restores $(thing)Saturation$(), which can be thought of as how full you are. Some foods are very filling, so they keep you from losing hunger for longer, while some foods have a short-lived effect.'),
+            text('You will lose hunger just from regular gameplay. Hunger drains faster if you do things like sprint, swim, or become $(l:mechanics/size_and_weight#overburdening)Overburdened$(). Information about the $(thing)Saturation$(), $(thing)Water$(), as well as $(thing)Nutrients$() available from a food is available by hovering over it in your inventory. Using $(item)$(k:key.sneak)$() reveals the full tooltip.'),
+            image('tfc:textures/gui/book/tutorial/food_tooltip.png', text_contents='How to display a food information tooltip.'),
+            text('All foods have a tooltip with this information. The tooltip includes the $(l:mechanics/decay)Decay Date$() of the food, which may be extended with preservation. When viewing this information, it\'s important to realize that not all foods are good to eat! For example, $(thing)Dough$() is a food, but it has no Nutrition, Saturation, or Water value. Eating it would not do much good.'),
+            image('tfc:textures/gui/book/gui/nutrition.png', text_contents='The nutrition screen, with bars showing the levels of each nutrient.'),
+            text('There are five nutrients, all obtainable from food: Fruit, Vegetables, Protein, Grain, and Dairy. Having a large amounts of all nutrients increases your maximum health, while having a poor nutrition decreases it. Eating a food gives you its nutrients. Meals such as $(l:mechanics/pot#soup)Soup$() combine more nutrients into one meal. This is important, because meals you ate a while ago don\'t count towards your nutrition.', title='Nutrients').anchor('nutrients'),
+            text('$(thing)Fruit$(): Fruit nutrients are mostly found from $(l:the_world/wild_fruits)Fruiting Plants$(), like berry bushes, and fruit trees. A notable exception to this is $(l:mechanics/crops#pumpkin)Pumpkins$() and $(l:mechanics/crops#pumpkin)Melons$(), which nutritionally are fruits.$(br)$(br)$(thing)Vegetables$(): Found in nearly every $(l:mechanics/crops)Crop$().$(br)$(br)$(thing)Protein$(): Protein can be gotten from killing $(l:the_world/wild_animals)Animals$() for meat. It can also be obtained from $(l:mechanics/crops#soybean)Soybeans$(), which have protein and vegetable nutrients.'),
```

Comment:
Reword: "Protein can be gotten by killing wild animals for meat" -> "Protein can be gotten from the meat of animals"